validator/accounts/cli_options.go

```diff
@@ -0,0 +1,74 @@
+package accounts
+
+import (
+	"github.com/prysmaticlabs/prysm/validator/accounts/wallet"
+	"github.com/prysmaticlabs/prysm/validator/keymanager"
+	"google.golang.org/grpc"
+)
+
+// Option type for configuring the accounts cli manager.
+type Option func(acc *AccountsCLIManager) error
+
+// WithWallet provides a wallet to the accounts cli manager.
+func WithWallet(wallet *wallet.Wallet) Option {
```

Comment:
Nice all the functional options

Reply:
:-) ya it is a super clean idea that I copied from Raul https://github.com/prysmaticlabs/prysm/pull/9881. I was reading about this for work a few days ago actually, and I think this is the original https://commandcenter.blogspot.com/2014/01/self-referential-functions-and-design.html. Very cool to see it in action!!